testing/trino-tests/src/test/java/io/trino/common/assertions/TrinoAssertions.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.common.assertions;
+
+import io.trino.spi.eventlistener.TableInfo;
+
+public class TrinoAssertions
+{
+    private TrinoAssertions() {}
+
+    public static TableInfoAssert assertThat(TableInfo actual)
+    {
+        return new TableInfoAssert(actual);
+    }
```

Comment:
Are you suggesting this only for the `TableInfoAssert`, for for all 4 of the methods in this class? (Note that the second commit adds 3 more methods here.)

I pulled everything into this one class to allow for a single static import, that then selects the correct assert based on the static type of the object:
```
import io.trino.common.assertions.TrinoAssertions.assertThat;
```
As opposed to:
```
import io.trino.common.assertions.TableInfoAssert.assertThat;
import io.trino.common.assertions.TableReferenceInfoAssert.assertThat;
import io.trino.common.assertions.BaseViewReferenceInfoAssert.assertThat;
import io.trino.common.assertions.FilterMaskReferenceInfoAssert.assertThat;
```
Especially if more fluent assertions are added in the future, this will provide a nifty entrypoint to discover all of the existing ones.

This is in line with how AssertJ itself works (see [Assertions](https://github.com/assertj/assertj/blob/main/assertj-core/src/main/java/org/assertj/core/api/Assertions.java)), and how they show adding custom assertions in their examples, e.g.: https://github.com/assertj/assertj-examples/blob/main/assertions-examples/src/test/java/org/assertj/examples/custom/UberSoftAssertions.java